### ISSUE TITLE
Register

### ISSUE THREAD

**Issue by yspk** [Pythonberg1997/notion-image-host#1]
@pythonberg1997 Hello!Our BlockBus Builder want to register Namelix's Bsc validator, Can I have your email or TG address?

**Comment by pythonberg1997** [comment]
You could get the info from here
https://github.com/bnb-chain/bsc-mev-info/tree/main/mainnet/validators

**Comment by yspk** [comment]
@pythonberg1997 Yes, we have configured Namelix's Bsc Validator on the builder side, but we received an Error of "builder not registered"
<img width="1074" alt="截屏2024-08-15 10 26 12" src="https://github.com/user-attachments/assets/624fe6dc-c525-4917-8b91-dfc6f8dd9375">
